frontend/src/api/network/secrets.ts

```diff
@@ -14,6 +15,13 @@ export const getSecret = (projectName: string, secretName: string): Promise<Secr
   });
 };
 
+export const getSecretsByLabel = (label: string, namespace: string): Promise<SecretKind[]> => {
+  return k8sListResource<SecretKind>({
```

Comment:
I am always wondering whether we could use `k8sListResourceItems` here instead of processing it, like what I did when fetching ImageStreams

Reply:
Yup, I understand, I thought of it. I think there is a problem with the `items` fetch -- I don't recall what it was, but I saw it crash under some circumstance that I don't recall. We'll need to do a lot of cleanup anyways, so I'll mark this down and investigate it when we get the other important stuff done.